scripts/servers.sh

```diff
@@ -199,7 +204,9 @@ new() {
     --key-name $keyname \
     --instance-type $instance_type \
     --security-groups "SSH + HTTP + HTTPS" \
+    --iam-instance-profile "avatar_upload_role"
```

Comment:
Why are you specifically mentioning Avatar upload here? Would it not make sense to have more general instance profiles like "production machine" that among others include the permission to upload avatars?

Reply:
Just created a prod_role with the correct permissions